### PR TITLE
7234 Upload blog assets to Azure

### PR DESCRIPTION
**Pull request by tomas-stefano**
I feel  a little out of shape after the eye surgery. 

First things first. 

* The fog does not support the azure (the fog-azure gem is not abstract to being used along with carrierwave and fog). So I will give it a go with the carrierwave-azure gem.
* I change the env variables names to a more abstract one. If you have better ideas, I feel open to suggestions. 

I am still unable to test because, as far as I know, I don't have access to Azure.

### CODE REVIEW

**Comment by Tr4pSt3R** [comment]
:+1:

File: .sample.env [hunk 1]
```diff
@@ -4,10 +4,10 @@ GA_API_EMAIL_ADDRESS=xx
 GA_PRIVATE_KEY_FILENAME=xx
 
 FOG_PROVIDER=xxx
-RACKSPACE_USERNAME=xxx
-RACKSPACE_API_KEY=xxx
-FOG_DIRECTORY=xxx
-RACKSPACE_CDN_HOST=xxx
+AZURE_ASSETS_STORAGE_ACCOUNT_NAME=xxx
```
Comment (misaka) [inline .sample.env]: I'm pretty sure we're going to actually go with `AZURE_ASSETS_STORAGE_BLOG_ACCOUNT_NAME` and `AZURE_ASSETS_STORAGE_BLOG_ACCOUNT_KEY` here. See https://github.com/moneyadviceservice/scripts/pull/585 and I'm doing a PR to add these variables to `/etc/mas/environment` in scripts.

**Comment by tomas-stefano** [comment]
@misaka changes done. Do you mind to re-review, please?

**Comment by munckymagik** [comment]
👍

**Comment by misaka** [comment]
:+1:

**Pull request merged**